src/planning_permission/utils/command_registry.py

```diff
@@ -74,7 +75,10 @@ def execute(self, command_string: str) -> str:
 
         try:
             parsed_args = command['parser'].parse_args(command_args)
-            return command['function'](**vars(parsed_args))
+            result = command['function'](**vars(parsed_args))
+            if asyncio.iscoroutine(result):
```

Comment:
Not sure if its necessary to use asyncio when we already are in an async function? but i might be wrong